packages/core/src/components/message/test/error/message.e2e.ts

```diff
@@ -0,0 +1,41 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/message/test/error/index.html';
+
+test.describe('tds-message', () => {
+  test('is rendered correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check if a component exists in the DOM */
+    const messageComponent = page.locator('tds-message');
+    expect(messageComponent).toBeTruthy();
+
+    /* Take screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('is in error variant', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const messageComponentDiv = page.locator('tds-message > div');
+    await expect(messageComponentDiv).toHaveClass(/error/);
+  });
+
+  test('has header text', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const messageComponent = page.locator('tds-message');
+    await expect(messageComponent).not.toBeEmpty();
```

Comment:
I think we have to test so that the tds-message have a div with class header that is not empty. Since if we remove the header from the component the tds-message component would still not be empty.

Reply:
Same for all other tests where you just check if "its not empty"

Reply:
Damn, I had it before but I lost it in copy/pasting, good check!

Reply:
Should be OK now. For subheader, I used toHaveText as in slot appears empty in the DOM as subheader text comes in between parenthesis of the component itself.